###Service to get the weight class of the spawned object

#true if we want to get the mass, initializes the get_mass_node
bool get_weight_class
---
#returns either "light","medium","heavy" as mass_class based on the knowledge from the ontology
string weight_type
string object_color